### PR TITLE
feature/CU-bf24ch_Initialize-project

### PR DESCRIPTION
**Pull request by Thurstag**

### CODE REVIEW

**Comment by Thurstag** [comment]
Task linked: [CU-bf24ch Initialize project](https://app.clickup.com/t/bf24ch)

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  extends: [
+    'plugin:vue/essential',
+    '@vue/standard'
+  ],
+  parserOptions: {
+    parser: 'babel-eslint'
+  },
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
```
Comment (Thurstag) [inline .eslintrc.js:14]: process.env.NODE_ENV === 'production' ? 'error' : 'warn'

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off'
```
Comment (Thurstag) [inline .eslintrc.js:15]: process.env.NODE_ENV === 'production' ? 'error' : 'warn'

File: package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    "chromedriver": "87",
+    "eslint": "^6.7.2",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "eslint-plugin-vue": "^6.2.2",
+    "geckodriver": "^1.20.0",
+    "vue-template-compiler": "^2.6.11"
+  }
```
Comment (Thurstag) [inline package.json:36]: On ne fixe pas la version de node ?

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+  </head>
+  <body>
+    <noscript>
+      <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
+    </noscript>
+    <div id="app"></div>
+    <!-- built files will be auto injected -->
```
Comment (Fantorem) [inline public/index.html:15]: Supprimer le commentaire

**Review by Fantorem — APPROVED** [review]

**Review by Tomansion — APPROVED** [review]

**Review by YesLife — APPROVED** [review]

**Pull request merged**